admin/admin.go

```diff
@@ -128,6 +133,13 @@ type Token struct {
 	Users    []User
 }
 
+type WebAuthnToken struct {
+	CredentialName string `json:"credential_name"`
+	DateAdded      int    `json:"date_added"`
```

Comment:
Can this be a `uint64` for consistency with the other `DateAdded` fields?

Reply:
Fixed!